services/app/src/components/base/QuestButton.tsx

```diff
@@ -46,6 +46,7 @@ export default class QuestButton extends React.Component<Props, {}> {
             {this.props.isOffline && <img className="inline_icon" src="images/offline_small.svg"/>}
             {quest.expansionhorror && <img className="inline_icon" src="images/horror_small.svg"/>}
             {quest.expansionfuture && <img className="inline_icon" src="images/future_small.svg"/>}
+            {quest.expansionwyrmsgiants && <img className="inline_icon" src="images/beast_small.svg"/>}
```

Comment:
We should probably have an actual icon for this...

Reply:
Suggestions welcome! Figure this would be ok, though, since the booster itself doesn't have a separate icon anywhere on it, and it's 100% Beast enemies, this seems about as intuitive as using a new / unrelated icon for it